json2html/jsonconv.py

```diff
@@ -78,88 +78,81 @@ def columnHeadersFromListOfDicts(self, inputtedJson):
                 -----------------------------
 
             @contributed by: @muellermichel
-        '''
+        """
 
-        if len(inputtedJson) < 2:
-            return None
-        if not isinstance(inputtedJson[0], dict):
+        if not inputted_json or not isinstance(inputted_json[0], dict):
```

Comment:
@Lekic Ah, now I get that line :)
The change ideally looks good.

Would be great if you could add a screenshot comparing the changes. This way, it helps in keeping track of changes.

Thanks for the patch. 👍 
Will be merging it for sure after this.